content/en/security_monitoring/detection_rules/security_monitoring.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Security Monitoring Rules
+further_reading:
+- link: "/security_monitoring/default_rules/"
+  tag: "Documentation"
+  text: "Configure default Security Monitoring rules"
+- link: "/security_monitoring/explorer/"
+  tag: "Documentation"
+  text: "Learn about the Security Signals Explorer"
+---
+
+## Create a New Rule
+
+To create a new Security Monitoring detection rule in Datadog, hover over **Security**, select **Security Rules**, and select the **New Rule** button in the top right corner of the page.
+
+## Rule Type
+
+For Security Monitoring, select **Log Detection** to analyze ingested logs in real time.
+
+## Choose a Detection Method
+
+### Threshold
+
+Define when events exceed a user-defined threshold. For example, if you create a trigger with a threshold of `>10`, a security signal occurs when the condition is met.
+
+### New term
+
+Detect when an attribute changes to a new value. For example, if you create a trigger based on a specific attribute, such as `country` or `IP address`, a security signal will be generated whenever a new value is seen which has not been seen before.
+
+## Define a Search Query
+
+{{< tabs >}}
+{{% tab "Threshold" %}}
+
+### Search query
+
+{{< img src="security_monitoring/detection_rules/threshold.png" alt="Define the search query" >}}
+
+Construct a search query using the same logic as a [log explorer search][1]. Each query has a label, which is a lowercase ASCII letter. The query name can be changed from an ASCII letter by clicking the pencil icon.
+
+Optionally, define a unique count and signal grouping. The defined group-by generates a signal for each group by value. Typically, the group by is an entity (e.g. user, IP, etc.). The group-by is also used to [join the queries together](#joining-queries).
+
+Add additional queries with the Add Query button.
+
+**Note**: The query applies to all Datadog events and ingested logs which do not require indexing.
+
+
+[1]: /logs/search_syntax/
+{{% /tab %}}
+
+{{% tab "New Term" %}}
+
+### Search query
+
+{{< img src="security_monitoring/detection_rules/new_term.png" alt="Define the search query" >}}
+
+Construct a search query using the same logic as a [log explorer search][1]. Each query has a label, which is a lowercase ASCII letter. The query name can be changed from an ASCII letter by clicking the pencil icon.
+
+Optionally, you can select a value to detect and define a signal grouping. The defined group-by generates a signal for each group by value. Typically, the group by is an entity (e.g. user, IP, etc.). The group-by is also used to [join the queries together](#joining-queries).
```

Comment:
for new term - you *must* select a new value to detect. You can *optionally* select a group by

Reply:
we should also cover what the learning duration setting is